vmm/src/config.rs

```diff
@@ -374,6 +374,8 @@ pub struct MemoryConfig {
     #[serde(default)]
     pub balloon_size: u64,
     #[serde(default)]
+    pub virtiomem_size: u64,
```

Comment:
Because you're adding a new field to the `MemoryConfig` structure, you must update the openapi YAML file to describe this field exists now. Additionally you should update the CLI to reflect this new parameter, as this might be a way for the user to specify the VM should be booted with a non-null `virtio-mem` region.

OR the other option might be to store the information about `virtio-mem` size from the `Vmm` itself, as this structure is not destroyed over a reboot. This would need to be handled cleanly as we don't want random fields to be added to the `Vmm` structure.

@sameo what do you think would be the cleanest approach?